src/ctl/src/cmd_impl/hummock/trigger_manual_compaction.rs

```diff
@@ -16,11 +16,15 @@ use risingwave_rpc_client::HummockMetaClient;
 
 use crate::common::MetaServiceOpts;
 
-pub async fn trigger_manual_compaction(compaction_group_id: u64) -> anyhow::Result<()> {
+pub async fn trigger_manual_compaction(
+    compaction_group_id: u64,
+    table_id: u32,
```

Comment:
why not pass a table array

Reply:
need to get internal_table_id for sst , so the input of client only table_id (outer), and get related internal_table_ids by fragment_manager in server